operator-framework-core/src/main/java/io/javaoperatorsdk/operator/api/config/DefaultInformable.java

```diff
@@ -6,14 +6,14 @@
 import io.javaoperatorsdk.operator.ReconcilerUtils;
 import io.javaoperatorsdk.operator.api.config.informer.InformerConfiguration;
 
-public class DefaultResourceConfiguration<R extends HasMetadata>
-    implements ResourceConfiguration<R> {
+public class DefaultInformable<R extends HasMetadata>
+    implements Informable<R> {
 
   private final Class<R> resourceClass;
```

Comment:
Do we need to duplicate this information? (I mean resource class). Wouldn't it be possible to always get it from informer?

Reply:
Ah yes, indeed, had intended to remove this and forgot. Thank you!

Reply:
Removed the class.